modules/apps/portal-settings/portal-settings-lang/src/main/resources/content/Language_fr.properties

```diff
@@ -112,7 +112,7 @@ please-enter-a-valid-ldap-base-dn=Veuillez saisir un DN de base de LDAP valide.
 please-enter-a-valid-ldap-search-filter=Veuillez entrer un filtre de recherche LDAP valide.
 please-enter-a-valid-ldap-server-name=Veuillez entrer un nom de serveur LDAP valide.
 please-enter-a-valid-locale=Veuillez entrer une locale valide svp.
-please-map-each-of-the-group-properties-group-name-and-user-to-an-ldap-attribute=Veuillez mettre en correspondance chaque propriété de groupe <em>Nom de groupe</em> et <em>Utilisateur</em> à un attribut LDAP.
+please-map-each-of-the-group-properties-group-name-and-user-to-an-ldap-attribute=Veuillez mettre en correspondance chaque propriété de groupe <em>Nom de groupe</em> et <em>Utilisateur</em> à un attribut LDAP.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.